### PR TITLE
Bw setup index

### PR DESCRIPTION
**Pull request by briankw23**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -1,5 +1,239 @@
...
+    iframe{
+        width: 374px;
+    }
+
+    #video-player{
+        margin: 0 auto;
+    }    
+ }
+
+>>>>>>> master
```
Comment (MakenHubbard) [inline main.css]: I guess delete this line ?

File: index.html [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+            </div>
+            <div id="right-col">
+
+            </div>
+        </main>
+        <footer>
+            <p>Copyright &copy; 2018 TeamTwerk</p>
+        </footer>
+    </body>
+>>>>>>> master
```
Comment (MakenHubbard) [inline index.html]: I guess delete this

File: main.css (hunk 1 above)
Comment (briankw23) [reply main.css]: Fixed

**Comment by MakenHubbard** [comment]
👍 👍 👍

**Pull request merged**